.github/workflows/main.yml

```diff
@@ -13,17 +14,29 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Build PostgreSQL container image
-      run: docker build --build-arg DATABASE_TYPE=postgresql --tag docker.pkg.github.com/mikecao/umami/umami:postgresql-$(echo $GITHUB_SHA | head -c7) .
+      run: | 
+        docker build --build-arg DATABASE_TYPE=postgresql \
+        --tag ghcr.io/$GITHUB_ACTOR/umami:postgresql-$(echo $GITHUB_SHA | head -c7) \
+        --tag ghcr.io/$GITHUB_ACTOR/umami:postgresql-latest \
```

Comment:
I've set a `latest` tag here for those who don't want to use pinned versions.

Reply:
At some point I'd like to stop pushing SHAs and start pushing the release number, e.g. `umami:postgresql-v1.0.0` instead of `umami:postgresql-f485498`